db/install.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<XMLDB PATH="mod/learningmap/db" VERSION="20211201" COMMENT="XMLDB file for Moodle mod/learningmap"
+<XMLDB PATH="mod/learningmap/db" VERSION="20230602" COMMENT="XMLDB file for Moodle mod/learningmap"
```

Comment:
Caution: this version is 20230601

Reply:
I think it's 20230619 here now (although version in install.xml doesn't mean anything for the upgrade process). When all PRs are merged I'd nevertheless bump versions to a current date (for release).
Fixed for now.